### ISSUE TITLE
Only allow historical countries before a country-specific cutoff, something submitted in 2010 shouldn't say "USSR"

### ISSUE THREAD

**Issue by corneliusroemer** [loculus-project/loculus#2836]

**Comment by theosanderson** [comment]
Idk if this is something for prepro/ingest or for post-hoc curation (even potentially with an automated script at some point).

**Comment by corneliusroemer** [comment]
Yeah not super important just something we can do to clean data. I think this is very much the remit of preprocessing though: checking that the input makes sense as early as possible - when there's a clear problem. We could of course also just warn as a first step.

**Comment by theosanderson** [comment]
When you mentioned 2010 I assumed you were talking INSDC data (but maybe I'm misinterpreting that and you are imagining that if Pathoplexus had launched in 2009). In terms of INSDC data, our stated aim has been that we ingest as-is and then create new versions with curatorial changes if required.

**Comment by corneliusroemer** [comment]
Yeah, I meant to warn/error on user submitted things like "Czech Republic" if it's collected post 2018 (or we just rewrite in this case as there's 1-1 mapping).
<img width="1086" alt="Brave Browser 2024-09-18 15 53 12" src="https://github.com/user-attachments/assets/23252e06-3a5d-4e28-8da5-b0e0f312a758">

We'd want to error for USSR if collection is 2010, though.
